financial-connections/src/main/java/com/stripe/android/financialconnections/model/serializer/EnumIgnoreUnknownSerializer.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.financialconnections.model.serializer
```

Comment:
could we move it to stripe-core?

Reply:
sure!

Reply:
https://github.com/stripe/stripe-android/pull/5583/commits/28a77dd062299f94347060264127aee175fa59e5